openfecwebapp/templates/legal-doc-search-results.html

```diff
@@ -0,0 +1,64 @@
+{% extends "layouts/main.html" %}
+{% import 'macros/legal-search.html' as legal_search %}
+{% set document_type_display_name = 'Advisory Opinions' %}
+
+{% block title %}
+Searching {{ document_type_display_name }} for "{{ query }}"
+{% endblock %}
+
+{% block body %}
+<header class="page-header slab slab--primary">
+  <span class="page-header__title">Legal resources // {{ document_type_display_name }}</span>
```

Comment:
Either in this PR or separately, we should update this area to use the new breadcrumb pattern: 
![image](https://cloud.githubusercontent.com/assets/1696495/14871322/ee3f4918-0c95-11e6-988f-a44cc80d56b5.png)

(`/templates/macros/breadcrumbs.html`)